mod_network_of_terms.erl

```diff
@@ -39,36 +39,41 @@ manage_schema(_Version, Context) ->
     z_datamodel:manage(?MODULE, Datamodel, Context).
 
 -spec observe_search_query(#search_query{}, z:context()) -> #search_result{}.
-observe_search_query(#search_query{search = {terms, Args}} = _Query, _Context) ->
+observe_search_query(#search_query{search = {terms, Args}} = _Query, Context) ->
     #search_result{
         result = network_of_terms_client:find_terms(
             [z_convert:to_binary(S) || S <- proplists:get_value(sources, Args)],
-            z_convert:to_binary(proplists:get_value(text, Args))
+            z_convert:to_binary(proplists:get_value(text, Args)),
+            Context
```

Comment:
I am wondering what causes a translation record to be sent through this - is there a specific situation you observed/foresee?

Reply:
oh, no, I added the `Context` passing to `network_of_terms_client:find_terms` and not to the `z_convert` call here.

`network_of_terms_client:find_terms`  now needs `Context` only so that I can call `network_of_terms_acl:is_allowed` there and limit access to the actual logic of `find_terms`.

I have otherwise not changed `find_terms` nor what it does.

Reply:
Ah of course, I should have double checked the parens - got distracted by to_binary actually (to my surprise) having a version with Context. 🤦